popolo/tests/test_importer.py

```diff
@@ -998,6 +998,42 @@ def test_link_and_source_without_a_note(self):
             person.links.get().url,
             "http://example.org/a-link-without-a-note")
 
+    def test_remove_identifiers_even_if_none_specified(self):
+        existing_person = models.Person.objects.create(name='Algernon')
+        existing_person.identifiers.create(
+            scheme='popolo:person',
+            identifier="a1b2"
+        )
+        existing_person.identifiers.create(
+            scheme='ignorable',
+            identifier="some-data-we-care-nothing-for"
+        )
+        existing_person.identifiers.create(
+            scheme='preserve-me',
+            identifier="data-that-should-be-kept"
+        )
+        input_json = '''
+{
+    "persons": [
+        {
+            "id": "a1b2",
+            "name": "Alice"
+        }
+    ]
+}
+'''
+        data = json.loads(input_json)
+        importer = PopItImporter(id_prefix='popolo:')
+        importer.import_from_export_json_data(data)
+        self.assertEqual(models.Person.objects.count(), 1)
+        # Reget the person from the database:
+        person = models.Person.objects.get(pk=existing_person.id)
+        self.assertEqual(person.name, 'Alice')
+        self.assertSequenceEqual(
+            person.identifiers.order_by('scheme').values_list('scheme', 'identifier'),
+            [(u'popolo:person', u'a1b2')]
+        )
+
 
```

Comment:
I wonder if there should be a near identical test with identifiers set to [] to show that they do the same thing?